### PR TITLE
Updated OnHold connection and Pushy id tracking

### PR DESCRIPTION
**Pull request by ArtemHryhorovGeniusee**

### CODE REVIEW

File: android/app/src/main/java/updated/mysterium/vpn/core/MysteriumAndroidCoreService.kt [hunk 1]
```diff
@@ -146,17 +146,13 @@ class MysteriumAndroidCoreService : VpnService(), KoinComponent {
                     currentState = state
                     if (previousState != currentState) {
                         when (currentState) {
-                            ConnectionState.ON_HOLD -> {
+                            ConnectionState.ONHOLD -> {
```
Comment (Sh1gure) [inline android/app/src/main/java/updated/mysterium/vpn/core/MysteriumAndroidCoreService.kt:149]: Not sure if it's a good change

File: android/app/src/main/java/updated/mysterium/vpn/model/manual/connect/ConnectionState.kt [hunk 2]
```diff
@@ -5,5 +5,5 @@ enum class ConnectionState(val state: String) {
     CONNECTING("Connecting"),
     CONNECTED("Connected"),
     DISCONNECTING("Disconnecting"),
-    ON_HOLD("OnHold")
+    ONHOLD("OnHold")
```
Comment (Sh1gure) [inline android/app/src/main/java/updated/mysterium/vpn/model/manual/connect/ConnectionState.kt:8]: Like here you divide words with camel case

File: android/app/src/main/java/updated/mysterium/vpn/core/MysteriumAndroidCoreService.kt (hunk 1 above)
Comment (ArtemHryhorovGeniusee) [reply android/app/src/main/java/updated/mysterium/vpn/core/MysteriumAndroidCoreService.kt]: Fix

File: android/app/src/main/java/updated/mysterium/vpn/model/manual/connect/ConnectionState.kt (hunk 2 above)
Comment (ArtemHryhorovGeniusee) [reply android/app/src/main/java/updated/mysterium/vpn/model/manual/connect/ConnectionState.kt]: Fix

**Review by Sh1gure — APPROVED** [review]

**Pull request merged**